charts/oes/values.yaml

```diff
@@ -332,8 +332,8 @@ gate:
   ##
   image:
     registry: quay.io/opsmxpublic
-    repository: ubi8-oes-gate
-    tag: v3.8.0
+    repository: ubi8-gate
```

Comment:
why did we change this?

Reply:
it's common gate; as per discussion with Jay & developer we have removed the  OES from image name.

Reply:
Don't we need any other changes in the helm chart to accomodate common gate?